webnode/package.json

```diff
@@ -9,7 +9,7 @@
     "chalk": "1.1.3",
     "css-loader": "0.28.7",
     "curl.lib.js": "^1.0.22",
-    "datamap-generator": "git+https://github.com/oysterprotocol/datamap-generator.git",
+    "datamap-generator": "^1.0.6",
```

Comment:
We should change this back.  What we had before would use whatever was in the master branch of the repo, regardless of npm version.

Reply:
Unless there is some very performant reason not to do it that way.

Reply:
no, this was due to npm publish, i will fix it. With this though, is there any reason to install from npm???